src/picobox/_box.py

```diff
@@ -185,3 +185,54 @@ def wrapper(*args, **kwargs):
                 return fn(*args, **kwargs)
             return wrapper
         return decorator
+
+
+class ChainBox(Box):
+    """ChainBox groups multiple boxes together to create a single view.
+
+    ChainBox for boxes is essentially the same as
+    :class:`~collections.ChainMap` for mappings. It mimics :class:`Box`
+    interface and hence can substitute one while provides a way to lookup
```

Comment:
nit: s/while provides a way to lookup for a dependency/, but also provides a way to look up dependencies